prow/cluster/jobs/istio/test-infra/istio.test-infra.master.yaml

```diff
@@ -68,19 +68,19 @@ presubmits:
 
   - name: flakey-run
```

Comment:
Why does test-infra need a flakey run? Can't we just fix the flakes and delete this job?